README.md

```diff
@@ -94,6 +94,28 @@ RSpec/FilePath:
     - spec/my_poorly_named_spec_file.rb
 ```
 
+## Non-goals of RuboCop RSpec
+
+### Enforcing `should` vs. `expect` syntax
+
+Enforcing:
+
+```ruby
+expect(calculator.compute(line_item)).to eq(5)
+```
+
+over:
+
+```ruby
+calculator.compute(line_item).should == 5
+```
+
+Is a feature of RSpec itself – here you can read about how to enable/disable one or the other: https://relishapp.com/rspec/rspec-expectations/docs/syntax-configuration#disable-should-syntax
+
+Is preferable to configure RSpec to not do any monkey patching at all (no `should` method on every object, and the outer-most `describe` or `context` must be called directly on RSpec): https://relishapp.com/rspec/rspec-core/v/3-7/docs/configuration/zero-monkey-patching-mode#monkey-patched-methods-are-undefined-with-%60disable-monkey-patching!%60
+
+Before disabling `should` you will need all your specs to use the `expect` syntax for them to work. You can use [Transpec](http://yujinakayama.me/transpec/), which will do the conversion for you.
```

Comment:
If you take my subheading suggestion from above, i think this should go into the previous section. Thanks for thinking to include this (I was mentally preparing to comment that we should mention Transpec but you were way ahead of me).

nit: "for them to work" is probably not necessary to communicate your point. fine to leave it if you prefer though.